packages/DropdownMenu/src/components/LinkItem/LinkItem.js

```diff
@@ -2,27 +2,32 @@ import React from "react";
 import PropTypes from "prop-types";
 import linkItemStyles from "../../Item.styles";
 
-const { bool, node, string } = PropTypes;
+const { bool, node, string, func } = PropTypes;
 
 const propTypes = {
   /** HTML for each LinkItem */
   children: node.isRequired,
   /** The url for the href */
   link: string.isRequired,
+  /** Callback to be executed when key is pressed */
+  onKeyDown: func,
   /** Should the link open content in a new tab */
   isExternal: bool,
 };
 
 const defaultProps = {
   isExternal: false,
+  onKeyDown: () => {},
 };
 
 const LinkItem = props => {
-  const { children, isExternal, link, ...moreProps } = props;
+  const { children, onKeyDown, isExternal, link, ...moreProps } = props;
 
   const linkItemProps = {
     role: "menuitem",
+    className: "dropdown-item",
```

Comment:
This is pretty much the same as what is being applied to the `Item.js` component. I think we should perhaps consolidate much of the code between `Item.js` and `LinkItem.js` so  we keep it DRY.